appinterface/projection/crossfire/crossfire.go

```diff
@@ -581,9 +587,28 @@ func (projection *Crossfire) computeCommitmentRank(
 		return fmt.Errorf("error getting validators' %s number from stats %v", crossfireValidatorStatsPrefix, err)
 	}
 
+	// filter out validators that not on list and missed taskSetup
 	var participatedValidatorCommits []view.CrossfireValidatorsStatsRow
 	for _, validatorCommit := range validatorCommits {
 		statsOperatorAddress := strings.Split(validatorCommit.Key, constants.DB_KEY_SEPARATOR)[1]
+
+		// check whether validator missed taskSetup or not
+		validatorRow, err := crossfireValidatorView.FindBy(view.CrossfireValidatorIdentity{
+			MaybeOperatorAddress: &statsOperatorAddress,
+		})
+		if errors.Is(err, rdb.ErrNoRows) {
+			// no validator found by tendermint address
+			continue
+		}
+		if err != nil {
+			return fmt.Errorf("error getting validator with operator addr %s %v", statsOperatorAddress, err)
```

Comment:
This will revert the whole process and will stop. Is that intended?

Reply:
Yes, should be~ if the error is non-norow error, then it's a big problem and should be fixed by us